src/main/resources/application.properties

```diff
@@ -18,6 +18,8 @@ rbac/mp-rest/url=https://ci.cloud.redhat.com
 rbac/mp-rest/connectTimeout=2000
 rbac/mp-rest/readTimeout=2000
 
+notifications-backend/mp-rest/url=https://ci.cloud.redhat.com
```

Comment:
Is this okay? I thought we could put in this value here and add the respective url depending on the environment stage/prod, .. via environment variable.

Reply:
I would replace `https://ci.cloud.redhat.com` which doesn't work anywhere with `http://localhost:8085` which is the dev host for notifications.

Reply:
FYI the env var no longer work on stage because notifications-gw is now deployed with Clowder. It still works on prod for now.

notifications-gw will require something like this on stage: https://github.com/RedHatInsights/policies-ui-backend/blob/master/src/main/java/com/redhat/cloud/policies/app/ClowderConfigInitializer.java

Reply:
the clowder property isn't present in the application. properties. I guess it's getting injected by clowder itself?